packages/opentelemetry-instrumentation-writer/tests/cassettes/test_chat/test_writer_async_chat_with_events_with_content.yaml

```
@@ -0,0 +1,73 @@
+interactions:
+- request:
+    body: '{
+        "model": "palmyra-x4", 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)